js/interactive-guides/circuit-breaker/circuit-breaker-callback.js

```diff
@@ -34,7 +34,7 @@ var circuitBreakerCallBack = (function() {
                         setTimeout(function () {
                             var stepWidgets = stepContent.getStepWidgets(stepName);
                             stepContent.resizeStepWidgets(stepWidgets, "pod", true);
-                            contentManager.setPodContentWithRightSlide(stepName,
+                            contentManager.setPodContentWithSlideDown(stepName,
```

Comment:
How about the other setPodContentWithRightSlide calls?

Reply:
According to Jay's design, the results pane should drop in from under the browser widget in the checkBalance step. In the other steps, the pod already there and it doesn't look like it slides down from the video. I wasn't sure if this is the intention or we want to keep it consistent regarding the slide down.

Reply:
Thanks, I'll approve the changes/